policyengine_us/variables/gov/states/md/tax/income/credits/ctc/md_non_refundable_ctc.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class md_non_refundable_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Maryland non-refundable Child Tax Credit"
+    definition_period = YEAR
+    unit = USD
+    documentation = "Maryland Child Tax Credit"
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-751-effective-until-712026-tax-credit-for-qualified-child"
+    defined_for = StateCode.MD
+
+    def formula(tax_unit, period, parameters):
+        md_ctc = tax_unit("md_ctc", period)
```

Comment:
move these inside the `if` block so we can improve efficiency by skipping querying them when they don't apply

Reply:
remove them from here